IFC4x3/PropertyEnumerations/i/PEnum_InputSignalType.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Necessary to have PEnum_InputSignalType when PEnum_InputOutputSignalType has the same content?

Reply:
Should be merged into one PEnum.